index.html

```diff
@@ -0,0 +1,166 @@
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title> RostyslavL </title>
+        <link rel="stylesheet" href="style.css">
+        <link 
+            rel="stylesheet" 
+            href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css" 
+            integrity="sha512-+4zCK9k+qNFUR5X+cKL9EIR+ZOhtIloNl9GIKS57V1MyNsYpYcUrUeQc9vNfzsWfV28IaLL3i96P9sdNyeRssA==" 
+            crossorigin="anonymous" 
+        />
+        <link rel="preconnect" href="https://fonts.gstatic.com">
+        <link href="https://fonts.googleapis.com/css2?family=Teko:wght@300;400;500;600;700&display=swap" rel="stylesheet">
+    </head>
+    <body> 
+        <div class="container">  
+            <div class="container-left">
+                <aside >
+                    <div class="photo" >
+                        <img src="img/me.jpg" alt="photo" >
+                    </div>
+                    <h1 class="name"> Rostyslav Lohvynov </h1>
+                </aside> 
+             <!-- Contact Section -->
+                <section class="section">
+                    <h2 class="contacts-title"> <i class="fas fa-address-book"></i>  Contacts </h2>
+                     <ul>
+                        <li class="contacts-item">
+                            <h3> Phone: </h3> <a href="tel:+48788249945" class="contacts-link"> +48 788 249 945 </a>
+                        </li>
+                        <li class="contacts-item">
+                            <h3> Email: </h3> <a href="mailto:rostyslavlogvinov@gmail.com" class="contacts-link"> rostyslavlogvinov@gmail.com</a>
+                        </li>          
+                    </ul>
+                    <div>
+                        <a href="https://github.com/RostyslavL"  class="social-media-link">
+                            <i class="fab fa-github"></i>
+                        </a>
+                        <a href="https://www.facebook.com/rostyslav.logvinov" class="social-media-link">
+                            <i class="fab fa-facebook"></i>
+                        </a>
+                        <a href="https://www.linkedin.com/in/rostyslav-l-815b271b0/" class="social-media-link">
+                            <i class="fab fa-linkedin-in"></i>
+                        </a>
+                    </div>
+                </section>
+            </div>
+            <div class="container-right"> 
+            <main>
+                    <!-- Education Section -->
+                        <section class="section">
+                            <h2 class="education-heading "> <i class="fas fa-user-graduate "></i>  Education </h2>
+                            <hr />
+                                <div>
+                                    <h2><i class="fas fa-university"></i>   Gdansk University of Technology, Gdansk </h2>
+                                    <details><summary> <strong> Master Degree </strong>   </summary>       
```

Comment:
I would rather avoid purely presentational tags like `strong` or `b` and will define a class instead